scripts/profiler/plot_nordic.py

```diff
@@ -561,6 +568,8 @@ def plot_events_from_file(
         if self.processed_events.tracking_execution:
             rects = []
             for ev in self.processed_events.tracked_events:
+                if ev.proc_start_time is None:
```

Comment:
Check processing end time too

Reply:
Actually in processed_events.py in function match_event_processing it is assured that event in tracked_events will either have proc_start_time and proc_end_time both set to None or both set to some value.